realm/realm-library/src/androidTest/java/io/realm/internal/android/ISO8601UtilsTest.java

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright 2015 FasterXML
+ * Copyright 2015 Realm Inc.
```

Comment:
That is the standard way of dealing with changes to code that is Apache 2 licensed. Basically it just says that FasterXML did the original implementation and Realm added some changes to it.